### PR TITLE
Revert "Update LICENSE"

### PR DESCRIPTION
**Pull request by mphetens**
Reverts mphetens/mphetens.github.io#5

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,5 @@
-# mphetens.github.io                  GNU GENERAL PUBLIC LICENSE
```
Comment (amethystens) [inline LICENSE:1]: 1 #mphetens.github.io
  GNU GENERAL PUBLIC LICENSE

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Comment by amethystens** [comment]
> Reverts mphetens/mphetens.github.io#5

**Review by amethystens — CHANGES_REQUESTED** [review]
naprawione

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

File: LICENSE
Comment (amethystens) [inline LICENSE:1]: 1

Comment (amethystens) [inline LICENSE:1]: LICENSE

**Review by amethystens — COMMENTED** [review]
LICENSE

**Review by amethystens — APPROVED** [review]

**Review by amethystens — CHANGES_REQUESTED** [review]
done

**Review by amethystens — APPROVED** [review]

**Comment by amethystens** [comment]
> Reverts mphetens/mphetens.github.io#5

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Comment by amethystens** [comment]
> Reverts mphetens/mphetens.github.io#5

**Comment by amethystens** [comment]
> Reverts mphetens/mphetens.github.io#5

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

File: LICENSE
Comment (amethystens) [inline LICENSE:1]: LICENSE

**Review by amethystens — APPROVED** [review]

**Review by Clydonight — APPROVED** [review]

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
-                       Version 3, 29 June 2007
```
Comment (amethystens) [inline LICENSE:2]: 2024

**Review by amethystens — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (amethystens) [inline LICENSE:1]: 1  +#  mphetens.github.io
  GNU GENERAL PUBLIC LICENSE

**Review by amethystens — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (amethystens) [inline LICENSE:1]: 1  +# mphetens.girhub.io
  GNU GENERAL PUBLIC LICENSE

**Review by amethystens — APPROVED** [review]

**Review by amethystens — APPROVED** [review]

**Pull request merged**